static/js/index.js

```diff
@@ -18,10 +97,61 @@ burger.addEventListener('click', () => {
     iwq.classList.add('show');
   }
 });
-// ====================================================================================
+// ============================================================================
+// eslint-disable-next-line no-unused-vars
+function detail(c) {
+  parpopup.style.display = 'flex';
+  const update = details.filter((item) => item.category === c);
+  parpopup__popup.innerHTML = `
+  <div class="head">
```

Comment:
- [ ] Kindly give your whole popup a margin/ padding top and bottom to ensure that the top and bottom sections of the popup do not touch the edges of the page but have a padding space between them and the page edges.

**This is how I am seeing your design right now:**

![image](https://user-images.githubusercontent.com/86318284/195418737-0bc3746c-608e-4fc3-ab8a-694bf1551dc6.png)

**This is the desired design:**

![image](https://user-images.githubusercontent.com/86318284/195418812-a6e936a8-75de-411c-90de-8cb4f42ce51b.png)

Reply:
Hi @Nemwel-Boniface, 👌
Thanks for your review.
I made the changes you stated above, so kindly check them and let me know if there are any issues
Have a nice day ☺️

Reply:
Good job @osamaashraf6 I see you made all the changes :clap: